dklearn/cross_validation.py

```diff
@@ -398,5 +412,5 @@ def train_test_split(*arrays, **options):
     if options:
         raise ValueError("Invalid parameters passed: %s" % str(options))
 
-    seed = different_seeds(1, random_state=random_state)[0]
+    seed, = random_state_data(1, random_state=random_state)
```

Comment:
Slight preference for the way it was before - on first glance the comma appears extraneous.